Formula/tarantool.rb

```diff
@@ -14,7 +14,10 @@ class Tarantool < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "autoconf"
```

Comment:
If these are needed only for head, please scope them inside a `head do` block

Reply:
I've tried to use the "head do" block, but the build of internal zstd library failed in this way.